rmgpy/data/thermoTest.py

```diff
@@ -75,6 +75,63 @@ def test_failing_loads_thermo_libraries(self):
         with self.assertRaises(Exception):
             database.load_libraries(os.path.join(path, 'libraries'), libraries)
 
+    def test_surface_thermo_libraries_have_surface_attributes(self):
+        """
+        Test that each entry in the surface thermo database has a 'metal' and 'facet' attribute, describing which metal
```

Comment:
Again, if we're testing that the whole database is well-formed, it should probably be in `RMG-Py/testing/databaseTest.py`. This file should be for testing that we can read/write/manipulate a well-formed database properly (much of which, this test does)